inventory/any.cf

```diff
@@ -184,6 +265,16 @@ bundle agent cfe_autorun_inventory_proc
       scope => "namespace";
 
   reports:
+    DEBUG|DEBUG_cfe_autorun_inventory_proc::
```

Comment:
I don't like to use verbose_mode alone because then its difficult to get just debugging output for an individual bundle. In its current state I also don't use verbose mode much since its hard to read.